torch/_inductor/decomposition.py

```diff
@@ -550,3 +550,12 @@ def select_decomp_table():
     if config.fallback_random:
         return decompositions
     return fast_random_decomps()
+
+
+@register_decomposition(aten.masked_scatter)
+def masked_scatter(self, mask, source):
+    if self.device.type == "cuda":
```

Comment:
Could you leave a comment here as well as to why do we just decompose CUDA but not CPU.